splinterd/src/daemon.rs

```diff
@@ -159,40 +160,78 @@ impl SplinterDaemon {
 
         let store_factory = create_store_factory(&self.db_url)?;
 
+        #[cfg(feature = "deprecate-yaml")]
+        {
+            let circuits_location = Path::new(&self.state_dir).join("circuits.yaml");
+            let proposals_location = Path::new(&self.state_dir).join("circuit_proposals.yaml");
+
+            let circuits_location_exists = circuits_location.exists();
+            let proposals_location_exists = circuits_location.exists();
+
+            if circuits_location_exists || proposals_location_exists {
+                error!("splinterd found yaml state files. run the `splinter upgrade` command to import your yaml state into a database");
```

Comment:
Suggest changing to "Found yaml state files; run the `splinter upgrade` command to import your yaml state into into a database"

Reply:
@vaporos I've also changed this code to attempt to convert the directory into an absolute path, similar to what @agunde406  suggested for the upgrade command PR